src/seq_mv/vector_batched.c

```diff
@@ -1185,8 +1185,8 @@ HYPRE_Int hypre_SeqVectorMassDotpTwo( hypre_Vector *x, hypre_Vector *y,
    {
       for (j = 0; j < k; j++)
       {
-         res_x = result_x[j];
-         res_y = result_y[j];
+         res_x = 0; //result_x[j];
```

Comment:
@rfalgout This initialization makes more sense to me. @PaulMullowney has confirmed the changes.

Reply:
Sounds good.